src/adp/azext_adp/aaz/latest/adp/account/data_pool/_create.py

```diff
@@ -0,0 +1,396 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "adp account data-pool create",
```

Comment:
If the data-pool sub group cmds are to be deprecated, please mark them as experimental in codegenv2 workspace and then export them.